articles/implementing-network-address-translation/index.md

```diff
@@ -0,0 +1,205 @@
+# Implementing Network Address Translation (NAT) on a CISCO router
+![hero Image](/engineering-education/implementing-network-address-translation/hero.jpg)
+[source](https://unsplash.com/photos/eVWWr6nmDf8)
+### Introduction
+This tutorial will cover how to implement NAT on a CISCO router, and it is expected that at the of the tutorial, the reader should learn the following:
+- What Network Address Translation is
+- Types of NAT
+- How to configure different types of NAT on a CISCO router.
```

Comment:
```suggestion
In this article, we will learn how to implement Network Address Translation (NAT) on a Cisco router. By the end of this article, the reader will learn what NAT is, different types of NAT and how to configure different types of NAT on a Cisco router.
```
A few suggestions:
- Always do not use the acronym, before using the abbreviation
- Use either `Cisco` or `CISCO` throughout the article. Consistency is important
- Add a prerequisite section
- Generally, adding bullet points in the introduction is not followed, instead write them as paras. Adding them in the summary would look good. (Just an opinion, I may be wrong)